### ISSUE TITLE
The order of loading model weights during pruning

### ISSUE THREAD

**Issue by BloomBabe** [david8862/keras-YOLOv3-model-set#181]
I noticed that during pruning my model shows too high values of loss functions, it turned out that the order of loading weights is important, and its necessary to first load the model  weights, and then warp layers in PruningWarp. I noticed that in EvalCallBack and get_yolo*_train_model this order is violated.

**Comment by david8862** [comment]
@BloomBabe  many thanks for info. I'll check and try to fix that later. The model pruning feature in this repo is out of maintained for quite a long time
